pkg/gatewayserver/gatewayserver_test.go

```diff
@@ -492,9 +625,10 @@ func TestGatewayServer(t *testing.T) {
 					upEvents[event] = ch
 				}
 				for _, tc := range []struct {
-					Name     string
-					Up       *ttnpb.GatewayUp
-					Forwards []int // Indices of uplink messages in Up that are being forwarded.
+					Name               string
+					Up                 *ttnpb.GatewayUp
+					StatusMsgSupported bool
```

Comment:
Why? `ptc` is available from outer scope right? That's also why I named it `ptc` to not conflict with `tc` here

Reply:
I think I misunderstood your comment that it should be only on inner table:
> The test code itself should just work with the test table entry

Fixed it to use the outer scope.